JSGames-master/js/monster.js

```diff
@@ -0,0 +1,71 @@
+let fallingspeed = 9.81;
+let angle;
+let power;
+let throwingDistance;
+let monsterDistance;
+let x = 0, y = 0, time = 0;
+
+function startGame () {
+    gameMenu.classList.remove('active');
+    inGame.classList.add('active');
+    setMonsterposition();
+}
+
+function setMonsterposition() {
+    monsterDistance = Math.round(Math.random() * 1000);
+    monster.style.left = monsterDistance + "px";
+    monsterDistanceBox.innerHTML = "The monster is " + monsterDistance + "m from you away.";
+}
+
+function selectWorld (world) {
+    startGame();
+    switch (world) {
+        case "earth":
+            fallingspeed = 9.81;
+            document.getElementsByTagName('main')[0].style.backgroundImage = "url(\"../img/monsterHunter/earth.jpg\")";
+            break;
+        case "moon":
+            fallingspeed = 1.62;
+            document.getElementsByTagName('main')[0].style.backgroundImage = "url(\"../img/monsterHunter/moon.jpg\")";
+            break;
+        case "mars":
+            fallingspeed = 3.69;
+            document.getElementsByTagName('main')[0].style.backgroundImage = "url(\"../img/monsterHunter/mars.jpg\")";
+            break;
+        case "jupiter":
+            fallingspeed = 24.79;
+            document.getElementsByTagName('main')[0].style.backgroundImage = "url(\"../img/monsterHunter/jupiter.jpg\")";
+            break;
```

Comment:
Wenn du etwas mit JS stylst, dann wird es inline in das HTML gesetzt.
Aus Sicht des HTMLs darfst du nicht mehr aus dem Ordner raus gehen um in den img reinzugehen (../) 
richtig wäre:
document.getElementsByTagName('main')[0].style.backgroundImage = "url(\"img/monsterHunter/earth.jpg\")";
Dann verschwinden auch die Fehlermeldungen